web/themes/custom/drevops/components/03-organisms/footer/footer.scss

```
@@ -0,0 +1,48 @@
+    @include ct-typography('text-small');
+
+    color: ct-color-dark('interaction-background');
+    letter-spacing: 0.08em;
+    text-decoration: none;
+
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+
+  @media (max-width: 767px) {
```

Stylelint (17.12.0)

[error] Expected "context" media feature range notation (media-feature-range-notation)

(media-feature-range-notation)